tests/test_flows_seed.py

```diff
@@ -89,42 +91,181 @@ def test_invalid_mnemonic(self):
         self.run_sequence(sequence)
 
 
-    def test_export_xpub_flow(self):
+    def test_export_xpub_standard_flow(self):
         """
-            Selecting "Export XPUB" from the SeedOptionsView should enter the Export XPUB flow and end at the SeedOptionsView.
+            Selecting "Export XPUB" from the SeedOptionsView should enter the Export XPUB flow and end at the MainMenuView
         """
+
+        def test_standard_xpubs(sig_tuple, script_tuple, coord_tuple):
+            self.run_sequence(
+                initial_destination_view_args=dict(seed_num=0),
+                sequence=[
+                    FlowStep(seed_views.SeedOptionsView, button_data_selection=seed_views.SeedOptionsView.EXPORT_XPUB),
+                    FlowStep(seed_views.SeedExportXpubSigTypeView, button_data_selection=sig_tuple[1]),
+                    FlowStep(seed_views.SeedExportXpubScriptTypeView, button_data_selection=script_tuple[1]),
+                    FlowStep(seed_views.SeedExportXpubCoordinatorView, button_data_selection=coord_tuple[1]),
+                    FlowStep(seed_views.SeedExportXpubWarningView, screen_return_value=0),
+                    FlowStep(seed_views.SeedExportXpubDetailsView, screen_return_value=0),
+                    FlowStep(seed_views.SeedExportXpubQRDisplayView, screen_return_value=0),
+                    FlowStep(MainMenuView),
+                ]
+        )
+            
         # Load a finalized Seed into the Controller
         mnemonic = "blush twice taste dawn feed second opinion lazy thumb play neglect impact".split()
         self.controller.storage.set_pending_seed(Seed(mnemonic=mnemonic))
         self.controller.storage.finalize_pending_seed()
 
+        # these are (constant_value, display_name) tuples
```

Comment:
Suggested edit:
```python
# These are lists of tuples: (constant_value, display_name)
```

You could define their types, but it hurts readability so the above comment would still be helpful.
```python
sig_types: list[tuple[str, str]] = SettingsConstants.ALL_SIG_TYPES
```

Reply:
I'll prefer your suggestion because expressive code > comment, and I'll leave the comment too.